Packs/CommonScripts/Scripts/CVECVSSColor/README.md

```diff
@@ -0,0 +1,25 @@
+## CVECVSSColor
+
+---
+This dynamic automation parses the CVSS score of a CVE and presents it in the layout in color according to its score.
+
+
+### Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | dynamic-indicator-section |
+| Cortex XSOAR Version | 6.8.0 |
+
+### Inputs
+
+---
+There are no inputs for this script.
+
+### Outputs
+
+---
+There are no outputs for this script.
```

Comment:
Please add the relevant human readable output

Reply:
There is none, its a dynamic script, you cant run it from a CLI :(